src/modules/ekf2/EKF/fake_pos_control.cpp

```diff
@@ -69,16 +71,20 @@ void Ekf::controlFakePosFusion()
 
 
 		const bool continuing_conditions_passing = !isHorizontalAidingActive();
+
 		const bool starting_conditions_passing = continuing_conditions_passing
+				&& ((Vector3f(getStateVariance<State::pos>()).xy().max() > pos_noaid_var)
```

Comment:
Need to consider fake_pos from beginning of flight as well as GPS lost in flight.